src/js/components/molecules/modal-button/modal-button.vue.html

```diff
@@ -1,6 +1,8 @@
-<button class="button" @click.prevent="triggerClick">
-    {{label}}
+<div class="modal-button__container">
+    <button class="button" @click.prevent="triggerClick">
```

Comment:
```suggestion
    <button class="button" @click="triggerClick" type="button">
```
Unless you actually want a submit button.

Reply:
👍